### PR TITLE
⏪📖 JW Player: Update documentation and remove configuration options

### PR DESCRIPTION
**Pull request by boyntoni**
**Description**

- This PR is required to accurately reflect JW Player AMP component usage.

### CODE REVIEW

**Comment by nainar** [comment]
@alanorozco could you help review this?

**Comment by boyntoni** [comment]
Hey @alanorozco, just wanted to follow up here. Hoping we can merge this in next week; if I need to do anything else, let me know.

**Comment by boyntoni** [comment]
Hi @alanorozco, wanted to follow up on this PR.

Thanks for your time,

Ian

**Comment by bobbyjw** [comment]
Hi @alanorozco, friendly bump!

File: extensions/amp-jwplayer/0.1/test/test-amp-jwplayer.js [hunk 1]
```diff
@@ -87,59 +87,41 @@ describes.realWin(
...
       });
       const iframe = jw.querySelector('iframe');
       expect(iframe).to.not.be.null;
       expect(iframe.tagName).to.equal('IFRAME');
       expect(iframe.src).to.equal(
-        'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html?search=dog&contextual=true&recency=9D&backfill=true'
+        'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html?search=dog&backfill=true'
       );
     });
     it('renders with a playlist and contextual, backfill parameters', async () => {
```
Comment (alanorozco) [inline extensions/amp-jwplayer/0.1/test/test-amp-jwplayer.js]: `contextual` has been removed, delete from test descriptions.

File: extensions/amp-jwplayer/amp-jwplayer.md [hunk 2]
```diff
@@ -91,19 +91,11 @@ Example:
...
-    <td>Denotes the type of the playlist. This is a search playlist that takes in a keyword or phrase as the search query and generates a playlist based on that search query. If contextual article matching is desired, use the value <code>__CONTEXTUAL__</code> (data-content-contextual must also be <code>true</code>).)</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-content-contextual</strong></td>
-    <td>Enables the Player to grab the OG title (or HTML title if there is no OG title) of a given webpage and use that as the search query (required to do Contextual Article Matching). (<strong>'true' or 'false'</strong>)</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-content-recency</strong></td>
-    <td>Limits the videos added into the playlist based on their age (i.e. the playlist includes videos that are max [xx] days old). Specified in the format <code>xD</code>, where x is a numerical value (i.e. <code>'9D'</code>).</td>
+    <td>Denotes the type of the playlist. If contextual article matching is desired, use the value <code>`__CONTEXTUAL__`</code>. If a search playlist is desired, input a keyword or phrase used to generate the search playlist.</td>
```
Comment (alanorozco) [inline extensions/amp-jwplayer/amp-jwplayer.md]: How does `__CONTEXTUAL__` still apply?

Comment (bobbyjw) [reply extensions/amp-jwplayer/amp-jwplayer.md]: The use of `__CONTEXTUAL__` is unchanged and should be passed back through `data-content-search`.  Previously, we required an additional field `data-content-contextual=TRUE` concurrently to enable the feature.  This additional requirement has now been removed for simplicity.

File: extensions/amp-jwplayer/0.1/test/test-amp-jwplayer.js (hunk 1 above)
Comment (boyntoni) [reply extensions/amp-jwplayer/0.1/test/test-amp-jwplayer.js]: This test is no longer valid @alanorozco , so I have just removed it.

**Comment by bobbyjw** [comment]
early christmas present - really appreciate this @alanorozco!

**Comment by bobbyjw** [comment]
Hello!  Wondering when we can expect this change to be live on the site: https://amp.dev/documentation/components/amp-jwplayer/

Our customers unfortunately keep incorporating `CONTEXTUAL` instead of `__CONTEXTUAL__` due to the original error caused by markdown.

**Pull request merged**